foundations/introduction/join_the_odin_community.md

```diff
@@ -84,7 +84,11 @@ Your Multiple Lines of Colorful Code
 
   1. First, create a free [GitHub account](https://github.com/join). As you will discover, GitHub is an integral part of the development workflow.
 
-  2. In a moment we're going to ask you to join the Discord community, here are some guidelines before you dive in: 
+  2. Now, Sign in to our [Discord server](https://discord.gg/fbFCkYabZB). Pop in and say hello! We've created an introductions room which is a great place to introduce yourself and we're always happy to welcome new community members. We have chat rooms for every development topic covered in our curriculum. Log into the chat and start exploring!
+  
+     * **Link your GitHub to your Discord profile:** `Discord Settings > Connections > Click GitHub icon > Allow Access > Make sure "Display on profile"` is toggled on. So others can see what you're working on and vice versa! 
```

Comment:
What would you think of something along the lines of the below tweak?

```suggestion
     * **Link your GitHub to your Discord profile:** Go to `Discord Settings > Connections`, then click the GitHub icon. In the new tab that opens click "Allow Access", then back in Discord make sure "Display on profile"` is toggled on. This will allow others to see what you're working on and vice versa! 
```

Reply:
Looks Great. Thanks for the suggestion. I'll commit it.